app/src/main/java/io/appium/uiautomator2/model/api/gestures/LongClickModel.java

```diff
@@ -0,0 +1,17 @@
+package io.appium.uiautomator2.model.api.gestures;
+
+import android.graphics.Point;
+
+import io.appium.uiautomator2.model.api.ElementModel;
+
+public class LongClickModel extends ElementModel {
+    public Double x;
+    public Double y;
+    public Double duration;
+
+    public LongClickModel() {}
```

Comment:
Codacy found an issue: [Avoid unnecessary constructors - the compiler will generate these for you](https://app.codacy.com/manual/dpgraham/appium-uiautomator2-server/pullRequest?prid=6222243)